Solutions/CohesitySecurity/Tests/az.test.py

```diff
@@ -116,7 +139,22 @@ def test_cohesity_restore_from_last_snapshot(self):
         )
         self.assertEqual(returncode, 0)
 
-        time.sleep(30)  # Sleep for 30 seconds
+        while True:
+            status = get_latest_playbook_run_status(
+                self.access_token,
+                self.subscription_id,
+                self.resource_group,
+                playbook_name,
+            )
+            if status != "Running":
+                break
+            time.sleep(5)  # Sleep for 5 seconds between status checks
+
+        assert (
+            status == "Succeeded"
+            or print(f"Assertion failed. Status: {status}. ")
```

Comment:
We should ideally print the entire response, not just status to aid debugging.

Reply:
that's why I have a big object return previously..
ok, then I need to get back the previous codes.

Reply:
done